main/app/headset/[headsetName]/page.tsx

```diff
@@ -0,0 +1,65 @@
+import { db } from '@/app/server/db/index'
+import { type Headset, headsets } from '@/app/server/db/schema/headsets'
+import { eq, SQLWrapper } from 'drizzle-orm'
```

Comment:
Database should not be accessed directly from Client. This should make use of  existing backend server routes